setup.py

```diff
@@ -57,7 +57,7 @@ def get_version(*file_paths):
         'django_safari_notifications',
     ],
     include_package_data=True,
-    install_requires=["django-model-utils>=2.0", "inquirer>=2.1"],
+    install_requires=["django-model-utils>=2.0", "inquirer>=2.1", "Pillow>=2.0.0"],
```

Comment:
If we do this, people will be forced to install Pillow even if they don't intend to use that manaemgent command. Please remove and make sure a clear message is shown to users when they use the command without PIL being installed